src/gui/vector/qgsfieldcalculator.h

```diff
@@ -31,25 +28,40 @@ class QgsVectorLayer;
  * \class QgsFieldCalculator
  *
  * \brief A dialog class that provides calculation of new fields using existing fields, values and a set of operators
+ *
+ * Sample usage of the QgsFieldCalculator class:
+ *
+ * \code{.py}
+ *     uri = "point?crs=epsg:4326&field=id:integer"
+ *     layer = QgsVectorLayer(uri, "Scratch point layer",  "memory")
+ *     layer.startEditing()
+ *     dialog = QgsFieldCalculator(layer)
+ *     dialog.exec_()
+ * \endcode
  */
 class GUI_EXPORT QgsFieldCalculator: public QDialog, private Ui::QgsFieldCalculatorBase
 {
     Q_OBJECT
   public:
     QgsFieldCalculator( QgsVectorLayer *vl, QWidget *parent = nullptr );
 
+    /**
+     * \brief Returns the field index of the field for which new attribute values were calculated.
+     *
+     * @return The field index if attribute values were calculated or -1, e.g. in case of geometry changes.
```

Comment:
```suggestion
     * \returns The field index if attribute values were calculated or -1, e.g. in case of geometry changes.
```

The various formatting scripts don't work with doxygen "@" notation